bananza-frontend/components/general/nav.js

```diff
@@ -0,0 +1,27 @@
+import Link from "next/link";
```

Comment:
I would add some more lateral padding to the buttons (Home, Profile etc.). Maybe something to highlight them too (some background, some outline?) - it's white text over some supposedly colored background.

Also, the Channel button may be obsolete (we agreed the profile is the same thing as a channel). Maybe we can call the profile as a channel on the page?

Reply:
Channel button has been removed.  Navbar has been improved somewhat, and may be modified more in the future.